src/Features/Core/Portable/Snippets/RoslynLSPSnippetItem.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Snippets
+{
+    internal readonly struct RoslynLSPSnippetItem
+    {
+        public readonly string? Identifier;
+        public readonly int Priority;
+        public readonly int? CaretPosition;
+        public readonly ImmutableArray<TextSpan> PlaceHolderSpans;
+
+        public RoslynLSPSnippetItem(string? identifier, int priority, int? caretPosition, ImmutableArray<TextSpan> placeholderSpans)
+        {
+            Identifier = identifier;
+            Priority = priority;
```

Comment:
Curious: Is Priority for ordering purposes with other snippets?

Reply:
added a comment about this, it's the number value that appears before the identifier in the lsp snippet